Commander/ArgumentDescription.swift

```diff
@@ -122,35 +122,52 @@ public class Options<T : ArgumentConvertible> : ArgumentDescriptor {
 
 public class Flag : ArgumentDescriptor {
   public typealias ValueType = Bool
-
+  
   public let name:String
+  public let falseName:String?
   public let flag:Character?
+  public let falseFlag:Character?
   public let description:String?
   public let `default`:ValueType
   public var type:ArgumentType { return .Option }
-
-  public init(_ name:String, flag:Character? = nil, description:String? = nil, `default`:Bool = false) {
+  
+  public init(_ name:String, falseName:String? = nil, flag:Character? = nil, falseFlag:Character? = nil, description:String? = nil, `default`:Bool = false) {
     self.name = name
+    self.falseName = falseName
     self.flag = flag
+    self.falseFlag = falseFlag
     self.description = description
     self.`default` = `default`
   }
-
+  
```

Comment:
Can you remove this whitespace (and the other random white space you have added)?